### DISCUSSION TITLE
how to send image via chat

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14809] [Wishlist]
I cant find attach file option in chat? is it available and if not then i think its such an imp feature to share images n file that chat should have,,,

**Comment by rskrzypczak** [comment]
Unfortunately, there is no such option.
It will be added at the request of: client or crowdfunding

**Reply by FranziskaSom** [reply]
what is the status here now because customers often leave things in the chat that don't belong there :)
deleting this every time via the database is a bit tedious ... what would be the costs because of the topic of crowdfunding. are there also other topics that people would like to implement. do you have a platform through which you manage this? we also have some topics/ideas in relation to project management that we would like to have changed ...
